docs/components/Toast/Toast.stories.mdx

```diff
@@ -18,10 +18,10 @@ function.
 
 When contributing to or consuming the Toast component, consider the following:
 
-- Keep text labels as short as possible.
+- Keep text labels as short as possible
   - They should be clear and concise, and should not take up more than one line.
-- Use the pattern: noun + verb.
-  - Examples: `Job saved`, `Quote approved`, `Client archived`.
+- Use the pattern "action + subject" to maintain [active voice](https://atlantis.getjobber.com/?path=/docs/content-voice-tone--page#active-voice)
```

Comment:
Thoughts:

- Any considerations on whether we could _only_ use action descriptors? (ie. Saved, Deleted, Dabbed) or make it optional?
- Any guidance on how specific the subject should be? (ie. Use generic terminology "Client" or specific terminology "Dolly Parton" where possible?)
- Can we shoutout that periods should not be used here? I've seen periods in toast that I think are unnecessary
- Any guidance on formatting? I noticed you're using sentence case and we have some direction that Jobber-specific subjects "Job, Client" should be capitalized but if toast needs to display something longer ("Emailed exported file" or something) then sentence structure feels better

Reply:
Cool :) am aligned with these comments!

Reply:
Updated! Please [take a look](https://update-toast-content-guideli.atlantis.pages.dev/?path=/docs/components-status-and-feedback-toast-docs--page#content-guidelines)